webapp/src/test/java/com/box/l10n/mojito/metrics/HierarchicalNameMapperWithCustomNamingConventionTest.java

```diff
@@ -0,0 +1,322 @@
+package com.box.l10n.mojito.metrics;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import io.micrometer.core.instrument.Meter;
+import io.micrometer.core.instrument.Tags;
+import io.micrometer.core.instrument.config.NamingConvention;
+import io.micrometer.core.instrument.util.HierarchicalNameMapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class HierarchicalNameMapperWithCustomNamingConventionTest {
```

Comment:
`public class` ?

why is the class named `HierarchicalNameMapperWithCustomNamingConventionTest`? usually the unit test that goes with a class `ClassA` would be called `ClassATest`. I don't see `HierarchicalNameMapperWithCustomNamingConvention`